docs/Model_Equations.rst

```diff
@@ -0,0 +1,152 @@
+.. math::        
+        E_i(t) \geq 0  \qquad  \forall t
+        
+.. math::
+        i \text{: asset}
+
+        a_i \text{: asset annuity [currency/kWp/year, currency/kW/year, currency/kWh/year]}
+
+        CAP_i \text{: asset capacity [kWp, kW, kWh]}
+
+        c_{var,i} \text{: variable operational or dispatch cost [currency/kWh, currency/L]}
+
+        E_i(t) \text{: asset dispatch [kWh]}
+
+The annual cost function of each asset includes the capital expenditure (investment cost) and residual value, as well as the operating expenses of each asset. It is expressed as follows:
+
+.. math:: 
+        a_i = \left( capex_i + \sum_{k=1}^{n} \frac{capex_i}{(1+d)^{k \cdot t_a}} - c_{res,i} \right) \cdot CRF(T) + opex_i
+.. math:: 
+        CRF(T) = \frac{d \cdot (1+d)^T}{(1+d)^t - 1}
+        
+.. math::
+        capex_i \text{: specific investment costs [currency/unit]}
+
+        n \text{: number of replacements of an asset within project lifetime T}
+
+        t_a \text{: asset lifetime [years]}
+
+        CRF \text{: capital recovery factor}
+
+        c_{res,i} \text{: residual value of asset i at the end of project lifetime T [currency/unit]}
+
+        opex_i \text{: annual operational and management costs [currency/unit/year]}
+
+        d \text{: discount factor}
+
+        T \text{: project lifetime [years]}
+
+The CRF is a ratio used to calculate the present value of the the annuity. The discount factor can be replaced by the weighted average cost of capital (WACC), calculated by the user. The lifetime of the asset t_a and the lifetime of the project T can be different from each other; hence, the number of replacements n is estimated using the equation below:
+
+.. math::
+        n = round \left( \frac{T}{t_a} + 0.5 \right) - 1
+        
+The residual value is also known as salvage value and it represents an estimate of the monetary value of an asset at the end of the project lifetime T. The MVS considers a linear depreciation over T and accounts for the time value of money by using the following equation:
+
+.. math::
+        c_{res,i} = \frac{capex_i}{(1+d)^{n \cdot t_a}} \cdot \frac{1}{T} \cdot \frac{(n+1) \cdot t_a - T}{(1+d)^T}
+
+
+Energy Balance Equation
+-----------------------
+
+One main constraint that the optimization model is subject to is the energy balance equation. The latter maintains equality between the incoming energy into a bus and the outgoing energy from that bus. This balancing equation is applicable to all bus types, be it electrical, thermal, hydrogen or for any other energy carrier.
+
+.. math::
+        \sum E_{in,i}(t) - \sum E_{out,j}(t) = 0 \qquad  \forall t
+
+.. math::
+
+        E_{in,i} \text{: energy flowing from asset i to the bus}
+
+        E_{out,j} \text{: energy flowing from the bus to asset j}
+
+It is very important to note that assets i and j can be the same asset (e.g., battery) however, one of the energy flowing values E_{in} or E_{out} should be zero at the same time step t.
```

Comment:
I do not understand this paragraph, especially about the flow being 0.

Reply:
the second part is a continuation of the first part. so if the assets i and j are the same asset (ex battery) one of the battery power flows should be zero (so either in or out). if you want, I can just remove the part starting with however *or* add in real life at the entire sentence

Reply:
Ah, you want to connect this to [this limitation](https://github.com/rl-institut/multi-vector-simulator/blob/dev/docs/Model_Assumptions.rst#infeasible-bi-directional-flow-in-one-timestep). However, this is what you would like the MVS to do, but not what it actually does, so I will remove the sentence.

Reply:
I actually did not remove, but cite the limitation instead: 8af22061d926b343198aeedbf509a0b009aaf273